base/array.jl

```diff
@@ -251,6 +251,14 @@ for (fname, felt) in ((:zeros,:zero), (:ones,:one))
         $fname(dims::Tuple) = ($fname)(Float64, dims)
         $fname(T::Type, dims...) = $fname(T, dims)
         $fname(dims...) = $fname(dims)
+
+        # #19635
+        function ($fname){T}(::Type{T}, arr::Array{T})
```

Comment:
One also needs to change a `@test_throws` when this is deleted. Thats why I put it here and not in `deprecated.jl`. You are still in favor of `deprecated.jl`?